lib/AST/ASTContext.cpp

```diff
@@ -2087,6 +2087,11 @@ TypeInfo ASTContext::getTypeInfoImpl(const Type *T) const {
     Align = 8;
     break;
 
+  case Type::Existential:
+    // '_Exists(T, struct Foo<T>)' gets erased to struct 'Foo<T>', so calculate
```

Comment:
Nit: the term 'erasure' may be unfamiliar to future compiler developers.  I suggest you change the phrase 'gets erased to struct 'Foo<T>' to 'is represented at runtime as 'struct Foo<T>'.  You could note parenthetically that this is an erasure strategy and that the use of T is restricted so that it doesn't affect runtime layout)

Reply:
Done